Makefile

```diff
@@ -267,7 +270,7 @@ $(OBJDIR)/%.o: $(CURDIR)/%.c include_dirs
 	@mkdir -p "$(@D)"
 	"$(WASM_CC)" $(WASM_CFLAGS) -MD -MP -o $@ -c $<
 
-include $(shell find $(OBJDIR) -name \*.d) /dev/null
+-include $(shell find $(OBJDIR) -name \*.d)
```

Comment:
I assume the `/dev/null` was there to prevent some kind of warning when no .d files are found?     Is there such a warning now?

Reply:
Probably not :) Unfortunately Windows bails on `/dev/null`, so something has to change about this. I'm open to suggestions for how to keep the warning.

Reply:
I meant that we *don't* want a warning there are no .d files yet generated.   At least thats the only reason I can see for adding this?  But I don't really know why it was added.   Dan do you remember why you added this originally?

Reply:
I don't think I had anything subtle in mind for this one; I just didn't think of `-include`.